app/src/main/java/com/xmartlabs/scasas/doapp/ui/TasksListFragment.java

```diff
@@ -116,7 +148,7 @@ private void updateTaskState(Task task) {
           @Override
           public void onSuccess(Task task) {
             int message = task.isFinished() ? R.string.good_job : R.string.cmon_you_can_do_this;
-            Snackbar.make(getView(), message, Snackbar.LENGTH_SHORT).show();
+            Toast.makeText(getContext(), message, Toast.LENGTH_SHORT).show();
```

Comment:
Cool.
You must check that the view is alive regardless. Use a compose and RxLifecycle